### DISCUSSION TITLE
Intercept question

### DISCUSSION THREAD

**Discussion by jaaronson006** [FrederickHuangLin/ANCOMBC#156] [General]
Hi!

I've been using your package to analyze the differential abundance between two sites (SiteA and SiteB) with the following code:
ancom.site <- ancombc2(
  data = all.physeq, 
  tax_level = "Phylum",
  fix_formula = "Site",
  p_adj_method = "fdr",
  pseudo = 0.5)

I'm confused by the output. There are columns called lfc_SiteB, diff_SiteB, etc. which I understand to be the log-fold change for Site B from Site A and other stats for Site A from Site B. However, I am also getting columns called lfc_(Intercept), diff_(Intercept), etc and I'm not sure what these mean. What intercept is this referring to and how does it differ from my baseline of Site A?

**Comment by FrederickHuangLin** [comment]
Hi @jaaronson006,

Since the ANCOM-BC2 methodology is based on a linear model framework, there will be an intercept term in the model. The interpretation is the same as the general linear model, which represents the grand mean of the data. Usually, the grand mean is not the parameter of interest, so you do not have to worry about the interpretations.

Best,
Huang
